docs/ml-guide.md

```diff
@@ -455,6 +459,17 @@ val pipeline = new Pipeline()
 // Fit the pipeline to training documents.
 val model = pipeline.fit(training)
 
+// now we can optionally save the fitted pipeline to disk
+model.save("/tmp/spark-logistic-regression-model")
+
+// we can also save this unfit pipeline to disk
+pipeline.save("/tmp/unfit-lr-model")
+
+// and load it back in during production
+val sameModel = Pipeline.load("/tmp/spark-logistic-regression-model")
+// or equivalently
```

Comment:
Just to be crystal clear, you would like me to remove 470 - 472?  The "equivalently" bit, right?